lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Requires parentheses around arrow function expressions with a single parameter.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireParenthesesAroundArrowParam": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * [1, 2, 3].map((x) => x * x);
+ * // params are always required for multiple parameters
+ * [1, 2, 3].map((x, y, z) => x * x);
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * [1, 2, 3].map(x => x * x);
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(options) {
+        assert(
+            options === true,
+            this.getOptionName() + ' option requires a true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'requireParenthesesAroundArrowParam';
+    },
+
+    check: function(file, errors) {
+        if (file.getDialect() !== 'es6') {
```

Comment:
> And I guess if we are using esnext by default later..

Well, i'm starting to doubt it would be a best course of action, since if we would have - 

```js
disallowKeywords: {
  es6: ['var']
}
```
and add it to the airbnb preset while enabling `esnext` by default - :boom: